gradle-plugin/src/main/kotlin/com/google/devtools/ksp/gradle/KspAATask.kt

```diff
@@ -76,7 +81,11 @@ abstract class KspAATask @Inject constructor(
             val sourceSetName = kotlinCompilation.defaultSourceSet.name
             val kspTaskName = kotlinCompileProvider.name.replaceFirst("compile", "ksp")
             val kspAADepCfg = project.configurations.detachedConfiguration(
-                project.dependencies.create("${KspGradleSubplugin.KSP_GROUP_ID}:symbol-processing-aa:$KSP_VERSION")
+                project.dependencies.create("${KspGradleSubplugin.KSP_GROUP_ID}:symbol-processing-aa:$KSP_VERSION"),
+                project.dependencies.create(
+                    "${KspGradleSubplugin.KSP_GROUP_ID}:symbol-processing-gradle-plugin:$KSP_VERSION"
```

Comment:
Why do you need this? It'll also fetch all runtime dependencies of KSP Gradle plugin and can be a surprise in the future.

Reply:
Also, I'd expect Kotlin stdlib and maybe reflection / etc to be listed here now that we're using `getPlatformClassLoader()`. If you're adding KSP Gradle plugin for this purpose, let's state the standard libraries explicitly. They should be available from Gradle and also KGP. Make sure you use either of them so that it doesn't end up downloading the third version.

Reply:
This is needed because we put the `KSPLoader` in gradle plugin code. `KSPGradleLogger` is used in `KSPLoader` so we need to add it here. Alternatively we can move all `KSPGradleLogger` away if we don't want KSP Gradle plugin in the classpath.

Reply:
Should we move KSPLoader into `kotlin-analysis-api` then? Probably into `KotlinSymbolProcessing`.